app/components/search/Sidebar/Sidebar.module.scss

```diff
@@ -35,6 +35,7 @@
     height: auto;
     width: 60%;
     padding: 0;
+    z-index: 998; // higher than everything except dropdown menu
```

Comment:
What do you think about moving this into a variable along with other z-index values? That also has the benefit of helping us discover a fixed set of indexes to avoid simply incrementing/decrementing relative to other values.